src/physics/cam/clubb_mf.F90

```diff
@@ -323,11 +375,14 @@ subroutine integrate_mf( nz,
      ! evaporation efficiency after Suselj etal 2019
      real(r8),parameter                   :: ke = 2.5e-4_r8
      !
-     ! fraction of rain detrained into downdrafts
-     real(r8),parameter                   :: fdd = 0._r8
+     ! height here downdrafts feel the surface
+     real(r8),parameter                   :: z00dn = 1.e3_r8
      !
-     ! fixed entrainment rate (debug only)
-     real(r8),parameter                   :: fixent = 2.e-4_r8
+     ! to fix entrainmnet rate
+     logical                              :: fixent = .false.

```

Comment:
Maybe these logical variables should be on the namelist (e.g., this one, do_condensation, etc). I understand that they are rarely changed but when they do, it requires editing clubb_mf.F90 and compiling the model again, right?

Reply:
yes, you need to re-compile if you change these values. I'm going to punt on making all these namelists, although I do agree they should be.